docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md

```diff
@@ -28,6 +29,255 @@
 | [v1.7.1](Data_Dictionary_Release_Notes.md#release-with-api-v171) | March 16, 2017 |
 | [v1.3.1](Data_Dictionary_Release_Notes.md#release-with-api-v131) | September 7, 2016 |
 
+## v3.4.4
+
+* __GDC Product__: GDC Data Dictionary
+* __Release Date__: July 24, 2025
+
+### New Features and Changes
+
+* Altered `demographic` Entity
+	* Changes made to `required`
+		* New permissible value: `sex_at_birth`
+		* Removed permissible value: `gender`
+	* Changes made to `deprecated`
+		* New permissible value: `gender`
+	* Removed permissible value: `weeks_gestation_at_birth`
+	* New property: `sex_at_birth`
+	* Removed property: `premature_at_birth`
+	* Removed property: `weeks_gestation_at_birth`
+	* Changes made to `population_group`
+		* New permissible value: `Unknown`
+		* New permissible value: `Not Reported`
+* Altered `family_history` Entity
+	* New property: `deprecated`
+	* Changes made to `properties`
```

Comment:
Anywhere you see this line, remove it, and then move all lines that were previously under it one tab to the left.